capellambse/model/crosslayer/fa.py

```diff
@@ -224,11 +224,12 @@ def owner(self) -> cs.PhysicalLink | cs.PhysicalPath | None:
     def exchange_items(
         self,
     ) -> c.ElementList[information.ExchangeItem]:
-        items = self.allocated_exchange_items
-        func_exchanges = self.allocated_functional_exchanges
-        assert isinstance(func_exchanges, cabc.Iterable)
-        for exchange in func_exchanges:
-            items += exchange.exchange_items
+
+        items = c.ElementList(self._model, [], information.ExchangeItem)
+        items.extend(self.allocated_exchange_items)
+        items.extend([item 
+            for fexch in self.allocated_functional_exchanges 
+            for item in fexch.exchange_items])
```

Comment:
Could you please look into setting up the [pre-commit hooks](https://pre-commit.com/) in our working copy? There's some formatting issues here that black would have taken care of. (I recognize we're missing a mention of that in this repo's CONTRIBUTING.)

For now, I've done this; and I've also changed the second part of this function back so it conforms to the Google style (only one `for` statement per comprehension).